manual/README.md

```diff
@@ -38,16 +38,16 @@ In the manual:
 
 - The PDF manual is in directory `texstuff` as file `manual.pdf`.
 
-Source files
+Source Files
 ------------
 The manual is written in an extended dialect of LaTeX and is split across many
 source files. During the build process, these source files are converted into
 classical LaTeX files using the tools available in the `manual/tools`
 directory. These files are then converted to the different output
-formats using either LaTeX or hevea.
+formats using either LaTeX or HeVeA.
 
 Each part of the manual corresponds to a specific directory, and each distinct
-chapters (or sometimes sections) are mapped to a distinct `.etex` file:
+chapter (or sometimes sections) are mapped to a distinct `.etex` file:
```

Comment:
Do you like this better? It's a little more concise, but what you have here is totally fine, too

Each part of the manual is organised into specific directories, with chapters or sections mapped to separate `.etex` files.